src/rangePicker/Day.tsx

```diff
@@ -45,12 +47,22 @@ export const Day = ({
   setSelectedDays,
   numberOfMonth,
   disabledBeforeToday,
+  disabledBeforeDate,
+  disabledAfterDate,
 }: Props) => {
+  if (disabledBeforeToday) {
+    const today = dayjs().format(FORMAT_DATE);
+    disabledBeforeDate =
+      disabledBeforeDate && dayjs(disabledBeforeDate).isAfter(today)
+        ? disabledBeforeDate
+        : today;
+  }
   let dateFormat = getDayFormat(day, jalali);
   const handleDisabledDate = () => {
     return (
       disabledDays.includes(dateFormat) ||
-      (disabledBeforeToday && dayjs(day).isBefore(dayjs().format(FORMAT_DATE)))
+      (disabledBeforeDate && dayjs(day).isBefore(disabledBeforeDate)) ||
```

Comment:
we need change this to handle for Jalali: 

`       (disabledBeforeDate && dayjs(dayjs(dateFormat)).isBefore(disabledBeforeDate)) ||
      (disabledAfterDate && dayjs(dayjs(dateFormat)).isAfter(disabledAfterDate))
`